packages/package-values.yaml

```diff
@@ -137,38 +137,6 @@ repositories:
         version: latest
         #! Gets replaced with imgpkg sha256 at build, this should be name:version
         sha256: "tkg-clusterclass-aws:latest"
-      - name: tkr-aws-machine-webhook
```

Comment:
Just curious, why are these packages being removed?

Reply:
This cleanup is way overdue: these have been dead for a while already.